tests/system/workspace_factory.py

```diff
@@ -51,6 +51,21 @@ class Package(object):
     <build_type>{build_type}</build_type>
   </export>"""
 
+        CATKIN_CMAKELISTS_TEMPLATE = """
+cmake_minimum_required(VERSION 2.8.3)
+project({name})
+find_package(catkin REQUIRED COMPONENTS {catkin_components})
+catkin_package()"""
```

Comment:
That's true, but at some point tests may just have to hand craft a `CMakeLists.txt` and not rely on a more and more complex function to generate everything for them.